app/src/main/java/com/example/playlistmaker/player/domain/PlayerUseCaseInter.kt

```diff
@@ -0,0 +1,41 @@
+package com.example.playlistmaker.player.domain
+
+class PlayerUseCaseInter(private val player: Player) {
```

Comment:
🍏 UseCase обычно отвечает за одну операцию и в нем есть только один метод execute. Этот класс больше похож на интерактор.

```suggestion
class PlayerInteractor(private val player: Player) {
```